tox.ini

```diff
@@ -15,6 +15,7 @@ usedevelop = True
 install_command = pip install -U {opts} {packages}
 deps =
   -r{toxinidir}/requirements-dev.txt
+  qiskit-terra
```

Comment:
I am thinking about adding a separate `requirements-extra.txt` in `docs_guidelines` folder and install both. I think that is a better way to manage this. Can you try that? When building in my local machine, I had an error for missing dependencies of `pylatexenc`. If that is really needed as well, can you add that to the `requirements-extra.txt`?

Reply:
I had just added ``pylatexenc`` to ``tox.ini`` and now everything works. If you prefer I can try to add the separate ``requirements-extra.txt`` and see how that goes

Reply:
It seems to work just fine